src/components/FormFields/index.js

```diff
@@ -139,7 +139,7 @@ class FormFields extends React.Component {
                 )
             }
         }
-        newEvents = sortBy(newEvents, (events) => (events.props.event.start_time))
+        // newEvents = sortBy(newEvents, (events) => (events.props.event.start_time))
```

Comment:
This fix has the (unintended) consequence that also the events returned by the recurring modal dialog remain unsorted. We should probably sort them when returning from the modal, but not sort any events added by hand.